packages/client/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "@blockful/client",
   "packageManager": "yarn@4.1.0",
   "scripts": {
-    "test": "vitest run",
-    "test:watch": "vitest",
+    "test:db:e2e": "hardhat test ./test/db.e2e.spec.ts --network localhost",
```

Comment:
this is enable the creation of integration tests for each gateway implementation